Revamp the landing page with cards (#6938)
sunpy/sunpy: docs/index.rst

sunpy core Documentation
************************

.. image:: tour.svg
  :align: center

``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.

> It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.

Reviewer: ```suggestion
``sunpy`` among other things provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
```
I think this is what Stuart wanted. Tho my corrections are bad. The sentence suggests we only do those three things.

Author: How about just changing "provides" to "includes"? That perhaps is a bit less exclusionary. Also, is there any major feature of the package we are missing with this description?

Author: I took another stab at it. Let me know what you think (and maybe this makes @Cadair a bit happier too)

.. grid:: 1 2 3 3
    :gutter: 2

    .. grid-item-card:: Tutorial
        :link: tutorial
        :link-type: ref
        :text-align: center

        :material-outlined:`accessibility_new;8em;sd-text-secondary`

        **New users start here!** Walkthrough on how to install sunpy and use the key features of the package.

    .. grid-item-card:: How-to Guides
        :link: how_to_guide
        :link-type: ref
        :text-align: center

        :material-outlined:`menu_book;8em;sd-text-secondary`

>         Snippets of code for accomplishing specific tasks with sunpy. Most useful for answering "How do I..." questions.

Reviewer: ```suggestion
        Snippets of code for accomplishing specific tasks with sunpy.
        Most useful for answering "How do I..." questions.
```
Hopefully this won't break.

    .. grid-item-card:: Example gallery
        :link: generated/gallery
        :text-align: center

        :material-outlined:`palette;8em;sd-text-secondary`

        Examples including plots on accomplishing common tasks using sunpy.

    .. grid-item-card:: Topic Guides
        :link: guide
        :link-type: ref
        :text-align: center

        :material-outlined:`school;8em;sd-text-secondary`

>         In-depth explanations of concepts and key topics. Most useful for answering "why" questions.

Reviewer: ```suggestion
        In-depth explanations of concepts and key topics.
        Most useful for answering "why" questions.
```

sunpy core Documentation
************************

.. image:: tour.svg
  :align: center

``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.

> It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.

Reviewer 2: I feel like this sentence is missing a "and a bunch of other stuff" line lol

Author: I think you commented on the wrong line and I'm not quite sure what you mean?

        :link-type: ref
        :text-align: center

        :material-outlined:`school;8em;sd-text-secondary`

        In-depth explanations of concepts and key topics. Most useful for answering "why" questions.

    .. grid-item-card:: Reference
        :link: reference
        :link-type: ref
        :text-align: center

        :material-outlined:`code;8em;sd-text-secondary`

        Technical description of the inputs, outputs, and behavior of each component of sunpy.

    .. grid-item-card:: Get Help
        :text-align: center

        :material-outlined:`live_help;8em;sd-text-secondary`

        .. button-link:: https://app.element.io/#/room/#sunpy:openastronomy.org
            :shadow:
            :expand:
            :color: warning

            **Join the chat**

        .. button-link:: https://github.com/sunpy/sunpy/issues
            :shadow:
            :expand:
            :color: warning

            **Report an issue**

        .. button-link:: https://community.openastronomy.org/c/sunpy/5
            :shadow:
            :expand:
            :color: warning

            **Post on Discourse**

.. button-link:: https://community.openastronomy.org/c/sunpy/5

>     Post on Discourse

Author: Ha oops. Do not know how that was left.

    .. grid-item-card:: How-to Guides
        :link: how_to_guide
        :link-type: ref
        :text-align: center

        :material-outlined:`menu_book;8em`

        Snippets of code for accomplishing specific tasks with sunpy. Most useful for answering "How do I..." questions.

    .. grid-item-card:: Example gallery
        :link: generated/gallery
        :text-align: center

        :material-outlined:`palette;8em`

        Examples including plots on accomplishing common tasks using sunpy.

    .. grid-item-card:: Topic Guides
        :link: guide
        :link-type: ref
        :text-align: center

        :material-outlined:`school;8em`

        In-depth explanations of concepts and key topics. Most useful for answering "why" questions.

    .. grid-item-card:: Reference
        :link: reference
        :link-type: ref
        :text-align: center

        :material-outlined:`code;8em`

        Technical description of the inputs, outputs, and behavior of each component of sunpy.

    .. grid-item-card:: Get Help
        :link: whatsnew
        :link-type: ref
        :text-align: center

>         :material-outlined:`support;8em`

Reviewer 3: ```suggestion
        :material-outlined:`live_help;8em`
```

        Find out how to join the chat room or post to discourse or GitHub to get help

    .. grid-item-card:: What's New
        :link: whatsnew
        :link-type: ref
        :text-align: center

        :material-outlined:`wb_sunny;8em`

        Find out what's new in sunpy

    .. grid-item-card:: Known Issues
        :link: known_issues
        :link-type: ref
        :text-align: center

>         :material-outlined:`wb_sunny;8em`

Reviewer 3: ```suggestion
        :material-outlined:`sentiment_dissatisfied;8em`
```
Heh...

        View a list of known issues or report new issues on our `GitHub issue tracker`_.

    .. grid-item-card:: Developer's Guide
        :link: dev_guide
        :link-type: ref
        :text-align: center

>         :material-outlined:`wb_sunny;8em`

Reviewer 3: ```suggestion
        :material-outlined:`construction;8em`
```
